### ISSUE TITLE
Brave with Firejail configuration doesn't connect

### ISSUE THREAD

**Issue by Robin-Sch** [keepassxreboot/keepassxc-browser#2133]
## Expected Behavior
When connecting to KeePassXC on Brave, it should work normally (like it does in Firefox)

## Current Behavior
Brave will try to connect, and even with KeePassXC already open in the background, it will open a new instance. After this it will still fail regardless of keeping the new instance open or closed.
Firefox works fine (doesn't open a new instance, connects immediately).

logs:
```
KeePassXC-Browser: Connecting to native messaging host org.keepassxc.keepassxc_browser
global.js:132 [Error client.js:403] KeePassXC-Browser - Failed to connect: Native host has exited.
global.js:132 [Error keepass.js:941] KeePassXC-Browser - No content script available for this tab.
global.js:132 [Error keepass.js:418] KeePassXC-Browser - 9: Key exchange was not successful.
global.js:132 [Error keepass.js:941] KeePassXC-Browser - No content script available for this tab.
global.js:132 [Error keepass.js:313] KeePassXC-Browser - 5: Cannot connect to KeePassXC. Check that browser integration is enabled in KeePassXC settings.
```

## Steps to Reproduce (for bugs)
1. Open KeePassXC with firejail
2. Open Firefox (with/without firejail) and try to connect -> works fine
3. Close Firefox
4. Open brave (with/without firejail) and try to connect -> opens new KeePassXC instance (without firejail?) -> fails to connect
5. Close newly opened instance of KeePassXC -> still fails
6. Close brave and repeat 4/5 but close old instance instead of new instance -> still fails
7. Close brave and all instances of KeePassXC and repeat 4/5 - -> opens new KeePassXC instance (without firejail? -> still fails to connect

## Debug info
KeePassXC - 2.7.6
KeePassXC-Browser - 1.9.0
Operating system: Linux
Browser: Brave

**Comment by droidmonkey** [comment]
So it works without the fire jail config set on Brave?

If so you'll need to debug your fire jail config.

**Comment by Robin-Sch** [comment]
I've set my firejail config on Brave, besides that, I also tried running Brave without firejail which gave the same result. Everything works fine with Firefox tho? See my edit repro steps.

**Comment by varjolintu** [comment]
https://github.com/keepassxreboot/keepassxc-browser/issues/1594

**Comment by Robin-Sch** [comment]
Again, I also tried running Brave without firejail which gave me the same result.

**Comment by varjolintu** [comment]
How is your Firefox or Brave installed? With .deb/Snap/Flatpak? Did you check the https://github.com/keepassxreboot/keepassxc-browser/wiki/Troubleshooting-guide ?

**Comment by Robin-Sch** [comment]
Yes I checked the Troubleshooting guide, for logs see my original issue. Both my browsers are installed using pacman, and KeePassXC is installed with the AppImage.

**Comment by Robin-Sch** [comment]
Yes, but it's weird that it works fine for Firefox, so I'm not really sure how to debug this.
Also, I tried the same thing on my laptop (browser without firejail, KeePassXC with firejail) which gives the same result, Firefox works fine but Brave opens a new instance.

I also just tried running KeePassXC without firejail. Now Brave complains about "Key exchange was not successful" (same logs as with firejailed), but does not open a new instance.
